GameObjectExtensions.cs

```diff
@@ -1,9 +1,15 @@
-﻿using UnityEngine;
+using UnityEngine;
 
 namespace DUCK.HieriarchyBehaviour
 {
 	public static class GameObjectExtensions
 	{
+		/// <summary>
+		/// Create a new GameObject as a child transform that has the given TBehaviour component.
```

Comment:
I'd phrase this in third person. describing what the function does rather than what you do by calling it. `Creates`, rather than `Create`